cmd/adapter/adapter.go

```diff
@@ -41,7 +42,6 @@ import (
 	"k8s.io/client-go/transport"
 	"k8s.io/component-base/logs"
 	"k8s.io/klog/v2"
-
```

Comment:
let's keep this line break, it makes the imports more readable

Reply:
Ack.